### DISCUSSION TITLE
MultiSelect dropdown custom colours for chip mode display

### DISCUSSION THREAD

**Discussion by demiahmed** [primefaces/community#803] [PrimeReact] [answered]
Hi I would like to propose a feature that allows us to add custom attributes to the Multi-Select dropdown options so we can give them custom colors. So far the all the options have the className of ```p-multiselect-token``` but would be great to pass in a ```color``` attribute along with the label name for custom colors for the chips.

Please correct me if I can already achieve this in some other way.

**Comment by melloware** [comment]
I think you will have to request this as an enhancement on GitHub issues.  I started a prototype of how this would work with PT options.

https://stackblitz.com/edit/react-35cda5?file=src%2FApp.js

**Comment by mertsincan** [comment]
Yes, PT with `token` key.

**Reply by demiahmed** [reply]
Is there any documentation on ```token```? can't seem to find on their website

**Reply by melloware** [reply]
@demiahmed I am using `token` in my reproducer here: https://stackblitz.com/edit/react-35cda5?file=src%2FApp.js the issue is there is no way to separate tokens.

**Comment by Real-Gecko** [comment]
So, currently there's no way to do it?

**Reply by melloware** [reply]
@Real-Gecko not without opening a GitHub Issue enhacement ticket requesting the feature.

**Reply by Real-Gecko** [reply]
I tried to create feature request in https://github.com/primefaces/primereact/issues/new/choose but was redirected here. Do I miss something?

**Reply by melloware** [reply]
You can click "Bug".

**Reply by Real-Gecko** [reply]
https://github.com/primefaces/primereact/issues/5935

**Reply by melloware** [reply, marked as answer]
Fixed in 10.5.1+ see: https://github.com/primefaces/primereact/issues/5935
